SPECS/kernel.spec

```diff
@@ -915,6 +916,12 @@ fi
 %{?_cov_results_package}
 
 %changelog
+* Tue Aug 08 2023 Gael Duperrey <gduperrey@vates.fr> - 4.19.19-7.0.17.1
+- Synced with hotfix XS82ECU1042
```

Comment:
Would ideally have been written "Security update based on XS82ECU1042". The line you copied from previous changelog entry was not a security update, that's what it only contained "Synced with hotfix...".